src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -202,6 +202,38 @@ def default_api_version(self):
         ResourceType.MGMT_IOTHUB: '2020-03-01',
         ResourceType.MGMT_ARO: '2020-04-30'
     },
+    '2020-09-01-hybrid': {
+        ResourceType.MGMT_STORAGE: '2019-06-01',
+        ResourceType.MGMT_NETWORK: '2018-11-01',
+        ResourceType.MGMT_COMPUTE: SDKProfile('2020-06-01', {
+            'resource_skus': '2017-09-01',
+            'disks': '2019-07-01',
+            'snapshots': '2019-07-01'
+        }),
+        ResourceType.MGMT_KEYVAULT: '2019-09-01',
+        ResourceType.MGMT_RESOURCE_LINKS: '2018-05-01',
+        ResourceType.MGMT_RESOURCE_LOCKS: '2016-09-01',
+        ResourceType.MGMT_RESOURCE_POLICY: '2016-12-01',
+        ResourceType.MGMT_RESOURCE_RESOURCES: '2019-10-01',
+        ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
+        ResourceType.MGMT_NETWORK_DNS: '2016-04-01',
+        ResourceType.MGMT_KEYVAULT: '2019-09-01',
+        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2016-09-01', {
+            'classic_administrators': '2015-06-01',
+            'policy_assignments': '2016-12-01',
+            'policy_definitions': '2016-12-01'
+        }),
+        ResourceType.DATA_KEYVAULT: '2016-10-01',
+        ResourceType.DATA_STORAGE: '2018-11-09',
+        ResourceType.DATA_STORAGE_BLOB: '2018-11-09',
+        ResourceType.DATA_STORAGE_FILEDATALAKE: '2018-11-09',
+        ResourceType.DATA_STORAGE_FILESHARE: '2018-11-09',
+        ResourceType.DATA_STORAGE_QUEUE: '2018-11-09',
```

Comment:
Hi @abbhar and @balansubr , may I know the supported api version storage blob, queue, file share service? from previous e-mail, the latest version supported should be 2019-07-07, right?

Reply:
Thanks for looking at this PR @Juliehzl . Yes you are correct. @abbhar was facing an issue while using that api version. He was going to reach out to you and CLI folks after doing some more investigations. I think the issue was , there was no Storage SDK found for that api version in the multi -pi package. @Abbhar could you pls elaborate the issue here

Reply:
I cleared cache and try with this version(2019-07-07). it's running fine. so looks like some issue in my local environment itself.  

However i want to confirm what version for ResourceType.DATA_STORAGE, we are planning to use in this profile ?

Reply:
DATA_STORAGE only support 2018-11-09. And with track2 SDK release, we will use DATA_STORAGE_BLOB/QUEUE/FILEDATALAKE for future control. 

Seems for dns, you could not use 2016-04-01 for some commands. Please check supported API version in your cloud env and api version which is used in CLI commands.

Reply:
@Juliehzl, for DNS we are planning to use  2016-04-01 there is no changes since the last profile in this. can you please help us to fix the DNS commands issue, so it can be loaded ?

Reply:
fix in https://github.com/Azure/azure-cli/pull/15456